analytics_data_api/v0/models.py

```diff
@@ -272,17 +312,25 @@ def get_users_in_course(
         elif ignore_segments:
             for segment in ignore_segments:
                 search = search.query(~Q('term', segments=segment))
-        # TODO: enable during https://openedx.atlassian.net/browse/AN-6319
-        # if cohort:
-        #     search = search.query('term', cohort=cohort)
+        if cohort:
+            search = search.query('term', cohort=cohort)
         if enrollment_mode:
             search = search.query('term', enrollment_mode=enrollment_mode)
         if text_search:
             search.query.must.append(Q('multi_match', query=text_search, fields=['name', 'username', 'email']))
 
-        # Sorting
-        sort_term = order_by if sort_order == 'asc' else '-{}'.format(order_by)
-        search = search.sort(sort_term)
+        # construct the sort hierarchy
+        sort_terms = []
+        for sort_policy in sort_policies:
+            sort_order = sort_policy['sort_order']
+            term = {
+                sort_policy['order_by']: {
+                    'order': sort_order,
+                    'missing': '_last' if sort_order == 'asc' else '_first',  # ordering of missing fields
+                }
+            }
+            sort_terms.append(term)
+        search = search.sort(*sort_terms)
```

Comment:
You could save a few lines with:

```python
search = search.sort(*[
    {
        sort_policy['order_by']: {
            'order': sort_policy['sort_order'],
            'missing': '_last' if sort_order == 'asc' else '_first'
        }
    }
    for sort_policy in sort_policies
])
```
Or something like it. I'm just a big fan of list comprehensions :) Just a nit though.